src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/ApplyExtensionRequest.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.digital.ho.hocs.casework.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class ApplyExtensionRequest {
```

Comment:
I have a slight concern that for some future tenant we might want an extension with a customisable period of days. Not sure if that is likely.

Reply:
The type refers to a type in casework.case_deadline_extension_type schema. This keeps a record of the type name and the number of days assigned to it. We can create new extension types by adding a record into this db with the name and the number of days.

Reply:
I do get that, I was just wondering if more generally there could be a type of extension where the number of days extended is not set in stone, but could be passed as a parameter. Probably that is unlikely so I reckon your approach is good.